hnn_core/gui/_viz_manager.py

```diff
@@ -0,0 +1,577 @@
+"""HNN GUI visualization management tool"""
+
+# Authors: Huzi Cheng <hzcheng15@icloud.com>
+
+import copy
+import io
+from functools import partial
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+from IPython.display import display
+from ipywidgets import (Box, Button, Dropdown, FloatText, HBox, Label, Layout,
+                        Output, Tab, VBox, link)
+
+from hnn_core.gui._logging import logger
+from hnn_core.viz import plot_dipole
+
+
+_dpi = 96
+_fig_placeholder = 'Run simulation to add figures here.'
+
+_plot_types = [
+    'current dipole',
+    'dipole@L2',
+    'dipole@L5',
+    'dipole@agg',
+    'input histogram',
+    'spikes',
+    'PSD',
+    'spectogram',
+    'network',
+    'layer-specific dipole',
```

Comment:
why do we have both layer-specific dipole and dipole@L2? My vote is to remove this one ... so clicking on "Plot" always goes to one axis. Currently "clear axis" doesn't work for this

Reply:
OK I'll remove `'layer-specific dipole'`